product_attribute_value_archive/models/product_attribute_value.py

```diff
@@ -69,12 +69,16 @@ def create(self, values):
         values_to_create = []
         unarchived_record_ids = set()
         for value in values:
+            attribute_id = value.get(
+                "attribute_id", self._context.get("default_attribute_id")
```

Comment:
```suggestion
                "attribute_id", self.env.context.get("default_attribute_id")
```

`_context` is deprecated for years.